### DISCUSSION TITLE
3.5" TFT LCD SHIELD FOR MEGA2560 not working in 8/16 bit parallel mode

### DISCUSSION THREAD

**Discussion by Stass48** [Bodmer/TFT_eSPI#3013] [General]
Hello! Here's the display:

![image](https://github.com/Bodmer/TFT_eSPI/assets/143362590/9ca2e9e8-8b70-4090-81cf-ecec303fd5be)

It works great with Arduino Mega 2560 and UTFT library:

![image](https://github.com/Bodmer/TFT_eSPI/assets/143362590/4fd80137-1da9-482b-8252-554d10aa1f95)

But I can't get this display to work with the RP2040. I need it to work in 8 bit mode. Performance is not as important as the number of pins on the Pico. I thought the display was designed to work in 16 bit mode, I tried it but that doesn't work either. I also didn't change any pins in the configurations, but it still doesn't work.

Display: 3.5" TFT LCD for Mega 2560 ILI9486

Actually, I'm trying to make a project in which there are 2 memory cards on two separate SPI buses, so using an SPI display is not acceptable for me :(

**Comment by Bodmer** [comment]
It is a 16 bit parallel display and will not work in 8 bit mode.

Use setup 105 or 106 and change just the display driver.

**Reply by Stass48** [reply]
It also not working in 16 bit parallel mode for me. But I think that a problem in frequency, may be... Else, I don't know, why it is not working.
